uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart1.tpl

```diff
@@ -0,0 +1,90 @@
+<div class="ac_results" id="crm-contact-actions-list">
```

Comment:
For all the class names we should follow BEM naming conventions.

Reply:
`crm__contact_action_menu`
 -`crm__contact_action_menu__user_info` - Left nav
   - any custom class inside this should named as `crm__contact_action_menu__<name>`
 - `crm__contact_action_menu__body`  - Right side body
   - any custom class inside this should named as `crm__contact_action_menu__<name>`